src/Files.App/Utils/Storage/Search/FolderSearch.cs

```diff
@@ -493,15 +493,11 @@ private async Task<ListedItem> GetListedItemAsync(IStorageItem item)
 			}
 			if (listedItem is not null && MaxItemCount > 0) // Only load icon for searchbox suggestions
 			{
-				var iconData = await FileThumbnailHelper.LoadIconFromStorageItemAsync(item, ThumbnailSize, ThumbnailMode.ListView, ThumbnailOptions.ResizeThumbnail);
-				if (iconData is not null)
-				{
-					listedItem.FileImage = await iconData.ToBitmapAsync();
-				}
+				var iconResult = await FileThumbnailHelper.GetIconAsync(item.Path, Constants.ShellIconSizes.Small, item.IsOfType(StorageItemTypes.Folder), false, IconOptions.ReturnIconOnly | IconOptions.UseCurrentScale);
```

Comment:
Split each parameter

```suggestion
				var iconResult = await FileThumbnailHelper.GetIconAsync(
					item.Path,
					Constants.ShellIconSizes.Small,
					item.IsOfType(StorageItemTypes.Folder),
					false,
					IconOptions.ReturnIconOnly | IconOptions.UseCurrentScale);

```